src/components/Comment/index.tsx

```diff
@@ -58,38 +136,64 @@ export class CommentBase extends React.Component<PublicProps> {
 
         <div
           // eslint-disable-next-line react/no-danger
-          dangerouslySetInnerHTML={sanitizeHTML(nl2br(comment.content), ['br'])}
+          dangerouslySetInnerHTML={sanitizeHTML(nl2br(initialComment.content), [
```

Comment:
I see that there's a test that covers `sanitizeHTML`, but no tests for either `nl2br`, nor the fact that `br` is being allowed via `sanitizeHTML`. Do you think we should have coverage for both of those?

Reply:
Good catch. I added a test for `nl2br`.